src/app/models/__tests__/form.server.model.spec.ts

```diff
@@ -1202,6 +1203,53 @@ describe('Form Model', () => {
         })
       })
     })
+
+    describe('updateEndPageById', () => {
+      it('should update end page and return updated form when successful', async () => {
```

Comment:
if you think it's important, we can also test that the correct defaults are inserted when we don't provide `title` and `buttonText`

Reply:
buttonText is already tested in this test case, but i'll add one for title too just in case